backend/src/test/java/com/github/nickklock/backend/controllers/UserControllerTest.java

```diff
@@ -59,6 +59,45 @@ void add_expect_status_created() throws Exception {
                         """)
         ).andExpect(status().isCreated());
     }
+
+    @Test
+    void add_expect_expect_400_because_email_fails() throws Exception {
```

Comment:
Zweimal expect in den folgenden Testnamen?

Reply:
pups